docs/src/architecture/08_concepts/signed_doc/docs/contest_delegation.md

```diff
@@ -74,7 +74,8 @@ considered.
 
 ## [COSE Header Parameters][RFC9052-HeaderParameters]
 
-No Headers are defined for this document.
+* [content type](../spec.md#content-type) = `None`
```

Comment:
I don't know what this means?
What are you implying by this.  The original wording is clear "There are no headers defined".  But something = None, means what?  Not descriptive enough for a specification.  Whats an implementer supposed to interpret that to mean?